### PR TITLE
Use correct service-name when using static service-discovery mechanism

### PR DESCRIPTION
**Pull request by fredfp**
references #898

### CODE REVIEW

**Comment by raboof** [comment]
OK TO TEST

File: runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala [hunk 1]
```diff
@@ -106,9 +106,8 @@ object GrpcClientSettings {
       case "static" | "grpc-dns" =>
         val host = clientConfiguration.getString("host")
         require(host.nonEmpty, "host can't be empty when service-discovery-mechanism is set to static or grpc-dns")
-        // Required by the Discovery infrastructure, even when we use static discovery.
-        if (serviceName.isEmpty)
-          serviceName = "static"
+        // Required by the Discovery infrastructure, set to host as we use static or grpc-dns discovery.
+        serviceName = host
```
Comment (raboof) [inline runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala:110]: should we make this `host:port` when the port is non-standard, or does netty take care of that for us?

Comment (fredfp) [reply runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala:110]: checking...

Comment (fredfp) [reply runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala:110]: I have no clue, sorry this is deeper than I can dig right now

Comment (raboof) [reply runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala:110]: :+1: - this is already a good improvement, thanks!

**Pull request merged**